### ISSUE TITLE
undefined is not an object (evaluating 'RNFreshchatSdk.FilterType')

### ISSUE THREAD

**Issue by yudiedrialviska** [baxitaxi/react-native-freshchat-sdk#2]
Hi Guys, after integrate on iOS, and I try running
but I found problem :
undefined is not an object (evaluating 'RNFreshchatSdk.FilterType')

any solution ?

**Comment by henrymoulton** [comment]
@yudiedrialviska did you find a fix for this? Having the same issue

"react-native-freshchat-sdk": "^0.5.7",

**Comment by yudiedrialviska** [comment]
Yes I already fix, I add Filter Type for object, because object can’t convert to String
You can follow my rpm package @yudiedrialviska/react-native-freshchat-sdk

**Comment by shivani1530** [comment]
Hi @yudiedrialviska , i am facing same issue "undefined is not an object (evaluating 'RNFreshchatSdk.FilterType')", could you tell me , how i do fix it step by step.?

**Comment by yudiedrialviska** [comment]
Open file /node_modules/@yudiedrialviska/react-native-freshchat-sdk/android/src/main/java/com/freshchat/consumer/sdk/react/RNFreshchatSdk.java

And at function jsonToBundle(), one case with bellow code :

case Map:
break;

**Comment by yudiedrialviska** [comment]
Open file /node_modules/react-native-freshchat-sdk/android/src/main/java/com/freshchat/consumer/sdk/react/RNFreshchatSdk.java

And at function jsonToBundle(), one case with bellow code :

case Map:
break;

**Comment by yudiedrialviska** [comment]
Open file /node_modules/react-native-freshchat-sdk/android/src/main/java/com/freshchat/consumer/sdk/react/RNFreshchatSdk.java

And at function jsonToBundle(), add one case with bellow code :

case Map:
break;

**Comment by shivani1530** [comment]
@yudiedrialviska thanks for the quik reply, but i got this error in iOS.

**Comment by edritech93** [comment]
what error ? @shivani1530

**Comment by shivani1530** [comment]
<img width="365" alt="Screenshot 2019-03-22 at 12 18 54 PM" src="https://user-images.githubusercontent.com/31620221/54805350-b6faa700-4c9c-11e9-9024-38aa06c64893.png">
@yudiedrialviska look at above screenshot.

**Comment by edritech93** [comment]
you can install this library, after I fix it : 
npm i @yudiedrialviska/react-native-freshchat-sdk

**Comment by edritech93** [comment]
and you can use import : 

import {
    Freshchat,
    FreshchatConfig,
    FreshchatUser,
    FreshchatNotificationConfig
} from '@yudiedrialviska/react-native-freshchat-sdk';

**Comment by shivani1530** [comment]
@yudiedrialviska i have already import this. :-(

**Comment by Espencerquinn** [comment]
@shivani1530 , did you ever figure out what was going on here? I'm currently running into this error while trying to integrate FreshDesk with React Native

**Comment by djsupersonicdesign** [comment]
anyone solve this problem on ios?

**Comment by djsupersonicdesign** [comment]
Hello?
